src/components/Base/Icon/index.jsx

```diff
@@ -12,7 +12,7 @@ export default class Icon extends PureComponent {
 
   render() {
     const { name, ...rest } = this.props;
-    const className = classnames('icon', `icon-${name}`, className);
+    const className = classnames('icon', `icon-${name}`, this.props.className);
```

Comment:
``const { name, className, ...rest } = this.props;``
``const classNames = classnames('icon', `icon-${name}`, className);``

Reply:
ok, but this change just fix eslint error.